README.md

```diff
@@ -4,31 +4,36 @@
 
-As your team onboards to Kardinal, you will be able to create flows with escalating levels of configurability:
+### Why choose Kardinal?
+- **Ephemeral Environments**: Spin up a new environment exactly when you need it, and just as quickly spin it down when you’re done.
+- **Minimal Resource Usage**: Only deploy the services you’re actively working on. Kardinal takes care of the rest, so you don’t waste resources.
+- **Flexible Flow Types**: Whether you need to test a single service or an entire application, Kardinal has you covered:
+    - Single-Service Flows: Perfect for when you’re tweaking just one service.
+    - Multi-Service Flows: Ideal for when your feature involves multiple services.
+    - State-Isolated Flows: Great for features that need their own databases or caches.
+    - Full Application Flows: For those times when you need end-to-end testing with full isolation.
+- **Cost Savings**: Kardinal can help you save big by avoiding unnecessary resource duplication. It’s a game-changer for teams looking to cut costs.
 
-1. **Single-service flows**: deploy a single service to test a new version of a service, sharing application state with all other flows
-2. **Multi-service flows**: deploy a set of services together to test a larger feature change involving multiple services, sharing application state with all other flows
-3. **State-isolated flows**: deploy a set of services together with new, isolated state (dbs, caches, queues) to test database migrations or other state-layer changes
-4. **Full application flows**: deploy an independent, full application with its own state layer for completely isolated end-to-end testing
+## Installation
 
-Even with isolated state and e2e full application flows, Kardinal will still deploy the absolute minimum resources necessary to test your changes. Isolation is done at the level of the request route, not by duplicating services in your cluster unnecessarily.
+### **Step 1: Install Kardinal**
+To install Kardinal, run the following command:
+```
+curl get.kardinal.dev -sL | sh
+```
+### **Step 2: Set up a development Kubernetes cluster**
 
-To see how Kardinal compares to other tools, and to get an idea of how would fit into your workflow, check out our [comparisons to alternatives](https://kardinal.dev/docs/references/comparisons).
+All you need is a Kubernetes cluster with Istio enabled, and kubectl installed on your machine, pointing to your cluster. If you need help with this, read more [here](https://kardinal.dev/docs/getting-started/install)
 
-Read more about Kardinal in our [docs](https://kardinal.dev/docs).
+### **Step 3: Deploy the Kardinal Manager to your cluster**
+Make sure that kubectl is pointing to your cluster, and then run the following command:
+```
+kardinal manager deploy kloud-kontrol
+```
 
```

Comment:
@tc87  we should put a link to go to the "Create your first flow" section of the docs at this point

Reply:
Done 👍